### ISSUE TITLE
Library managements

### ISSUE THREAD

**Issue by favonia** [RedPRL/sml-redprl#353]
I want a standard library so that I do not have to define `Id` everywhere.

**Comment by jonsterling** [comment]
That'd be nice. In JonPRL we had a way to build different files together, kind of like `sources.cfg` in Twelf. We could support something like that for RedPRL, which would enable us to design a standard library of definitions and tactics.

**Comment by jonsterling** [comment]
As for a more long-term vision, as part of #307 I intend to eventually have a SML-style module system, and also support something analogous to the ML Basis System for programming-in-the-large. But that's a very long-term project, so we shouldn't wait for it before addressing library management or importing in the current version of RedPRL.

**Comment by mortberg** [comment]
I also want this. Maybe as a first step implement something very simple so that one can just import files in the same directory? This should be a useful addition before the POPL tutorial so that users don't have to copy-paste code into their own files.

**Comment by jonsterling** [comment]
Yes, some very basic version of this would be good. I think, to start off, that we ought to at least have a version where you can pass multiple files to RedPRL and it will treat them in order as extensions.
